core/src/mindustry/world/blocks/distribution/ItemConveyor.java

```diff
@@ -59,7 +43,9 @@ private static int compareItems(long a, long b){
     @Override
     public void setStats(){
         super.setStats();
+
         stats.add(BlockStat.itemsMoved, speed * 60 / itemSpace, StatUnit.itemsSecond);
+        stats.add(BlockStat.boostEffect, "$blocks.itemsmoved");
```

Comment:
made it more clear what a boost effect does to conveyor blocks
(really confused me when i was a starting player that conveyors were affected as well)
and it makes the boost entry for the pull's conveyor less out of place